pkg/controllers/scheduling/controller.go

```diff
@@ -166,5 +166,6 @@ func (c *Controller) Register(_ context.Context, m manager.Manager) error {
 		Named("scheduling").
 		For(&v1.Pod{}).
 		WithOptions(controller.Options{MaxConcurrentReconciles: 10_000}).
+		WithLogger(zapr.NewLogger(zap.NewNop())).
```

Comment:
Just for my edification, why `NewNop` here?

Reply:
Controller runtime has a really annoying logging format and logs whenever you return an error. I wanted to use our kpkg logger but also leverage the backoff retry logic that controller-runtime implements on error.

Reply:
In the longer run, I think we need to figure out how to be in harmony with our controller framework.